packages/fast-components-react-base/src/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,93 @@
+import * as ReactDOM from "react-dom";
+import { BreadcrumbItemProps } from "../breadcrumb-item";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { BreadcrumbClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    BreadcrumbHandledProps,
+    BreadcrumbProps,
+    BreadcrumbUnhandledProps,
+} from "./breadcrumb.props";
+import { get } from "lodash-es";
+import * as React from "react";
+
+class Breadcrumb extends Foundation<
+    BreadcrumbHandledProps,
+    BreadcrumbUnhandledProps,
+    {}
+> {
+    public static displayName: string = "Breadcrumb";
+
+    protected handledProps: HandledProps<BreadcrumbHandledProps> = {
+        children: void 0,
+        seperator: void 0,
+        managedClasses: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLDivElement> {
+        return (
+            <nav
+                {...this.unhandledProps()}
+                aria-label="Breadcrumb"

```

Comment:
I don't think we want to hardcode this string in

Reply:
+2

Reply:
@eljefe223 unless this is always an accessibility requirement (I don't think it is), we should give guidance and leave this to be passed via unhandledProps

Reply:
According to this it is https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html

Reply:
It would still have to be localized I would think, which means that the user needs to pass the string value